stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -39,9 +39,9 @@
 ;; (define-constant ADDRESS_VERSION_P2SH 0x01)
 ;; (define-constant ADDRESS_VERSION_P2WPKH 0x02)
 ;; (define-constant ADDRESS_VERSION_P2WSH 0x03)
-(define-constant ADDRESS_VERSION_NATIVE_P2WPKH 0x04)
-(define-constant ADDRESS_VERSION_NATIVE_P2WSH 0x05)
-(define-constant ADDRESS_VERSION_NATIVE_P2TR 0x06)
+;; (define-constant ADDRESS_VERSION_NATIVE_P2WPKH 0x04)
+;; (define-constant ADDRESS_VERSION_NATIVE_P2WSH 0x05)
+;; (define-constant ADDRESS_VERSION_NATIVE_P2TR 0x06)
```

Comment:
Please remove commented code

Reply:
Should we remove all the constants? I understand that is part of the documentation

Reply:
These constants should all be defined in `pox-4.clar`.  Before, they were split between `pox-4-mainnet.clar` and `pox-4.clar`, but now it seems that they're all commented out?

Reply:
Yeah, these need to be uncommented.  All of these constants are required.

Reply:
@jcnelson where are they used? I only see it used in pox.clar
other pox contracts use MAX_ADDRESS_VERSION

Reply:
Yes, leave them in -- they're accessible via the RPC API